backend/tests/prisonerAlerts.test.ts

```diff
@@ -111,8 +140,6 @@ describe('prisoner alerts', () => {
       active: 'ACTIVE',
     }
 
-    res.locals.responseHeaders['total-records'] = 1
-
```

Comment:
this was cheekily pretending there was a single record, when the mock actually has none, so removed and adjusted the expectation below to "0" records